Content.Client/_DV/UserInterfaces/Systems/Chat/ChatUIController.cs

```diff
@@ -12,6 +15,12 @@ namespace Content.Client.UserInterface.Systems.Chat;
 
 public sealed partial class ChatUIController : IOnSystemChanged<CharacterInfoSystem>
 {
+    private ChatSelectChannel _currentChannel = ChatSelectChannel.None;
```

Comment:
Make public so this can be changed by the ChatBox.


```suggestion
    public ChatSelectChannel CurrentChannel = ChatSelectChannel.None;
```